File: src/main/java/com/hdjunction/domain/hospital/Hospital.java
package com.hdjunction.domain.hospital;

import com.hdjunction.domain.baseentity.BaseTimeEntity;
import lombok.AccessLevel;
import lombok.NoArgsConstructor;

import javax.persistence.*;

@Entity
@NoArgsConstructor(access = AccessLevel.PROTECTED)
public class Hospital extends BaseTimeEntity {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "hospital_id")
    private Long id;

    @Column(nullable = false)
    private String name;

    @Column(nullable = false)
    private String ykiho;

    @Column(nullable = false)
    private String directorName;
}

File: src/main/java/com/hdjunction/domain/hospital/Hospital.kt
package com.hdjunction.domain.hospital

import com.hdjunction.domain.baseentity.BaseTimeEntity
import javax.persistence.Column
import javax.persistence.GeneratedValue
import javax.persistence.GenerationType.IDENTITY
import javax.persistence.Id

class Hospital(
    @Id @GeneratedValue(strategy = IDENTITY) @Column(name = "hospital_id")
    var id: Long,

    @Column(nullable = false)
    val name: String,

    @Column(nullable = false)
    val ykiho: String,

    @Column(nullable = false)
    val directorName: String
) : BaseTimeEntity()